content/v2.1/en/installation/install-ks-on-linux-airgapped.md

```diff
@@ -1,51 +1,29 @@
 ---
-title: "Install KubeSphere on Air Gapped Linux Machines"
+title: "Air-Gapped Installation"
 keywords: 'kubernetes, kubesphere, air gapped, installation'
-description: 'How to install KubeSphere on air gapped Linux machines'
+description: 'How to install KubeSphere on air-gapped Linux machines'
 ---
 
-The air gapped installation steps are almost the same as the online installation. The air gapped installer will create a Docker local registry as the local image registry. We will demonstrate how to install KubeSphere 2.1.1 and Kubernetes on air gapped environment.
-
-Note: The dependencies in different operating systems may caused upexpected installation problems. If you encounter any installation problems on air gapped environment, please describe your OS information and error logs.
-
-> Important
-> - Please make sure there are 100G at least in your disk of the target machine
-> - Docker uses `/var/lib/docker` as the default directory where all Docker related files, including the images, are stored. Before loading the images into Docker. We recommend you to add additional storage to a disk mounted at `/var/lib/docker` and `/mnt/registry` respectively, it is necessary to prepare additional disk with 100G at least, you can use the [fdisk](https://www.computerhope.com/unix/fdisk.htm) command to prepare.
```

Comment:
"add additional storage to a disk mounted at `/var/lib/docker` and `/mnt/registry` respectively" is necessary to notify users before installation.